database/migrations/2020_09_28_073852_create_posts_table.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreatePostsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('posts', function (Blueprint $table) {
+            $table->increments('id');
+            $table->foreignId('user_id');
```

Comment:
this is a nice helper but sometimes you dont need bigInteger. And I think users migration is only integer and not bigInteger.
From the docs:
`$table->foreignId('user_id');` - Alias of `$table->unsignedBigInteger('user_id').`

Reply:
I dont say it's wrong, but please double check the users migration